### PR TITLE
Add plan and use-cases

### PR DESCRIPTION
**Pull request by KseniiaHorai**

### CODE REVIEW

File: docs/requirements/stakeholders-needs.md [hunk 1]
```diff
@@ -319,24 +356,35 @@
...
-        <td></td>
+        <td> 
+        - Користувач авторизований</br>
+        - Користувач є членом проєкту
+        </td>
     </tr>
      <tr>
         <td><b>Результат:</b></td>
-        <td></td>
+        <td>Завдання видалене</td>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]: You should display the result of the use case as noun.

Better use `Видалене завдання`.

File: docs/requirements/stakeholders-needs.md [hunk 2]
```diff
@@ -319,24 +356,35 @@
...
         <td>
+         - RejectedAcess_EXC - Користувач не має відповідного дозвола для видалення цієї задачі </br>
+         - DeleteTask_CancelButton_EXC - користувач натиснув кнопку "Відміна"
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          <ol>
+            <li>Користувач обирає проєкт і натискає на кнопку "Видалити задачу"</li>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]: You missed a step when user choose task to delete.

```
Користувач обирає проєкт
Користувач обирає задачу і натискає на кнопку "Видалити задачу"
```

File: docs/requirements/stakeholders-needs.md [hunk 3]
```diff
@@ -824,24 +872,37 @@
...
+        <td>
+         -Користувач багаторазово неправильно вводить пароль.<br/>
+         -Виявлення підозрілої активності в обліковому записі користувача.<br/>
+         -Користувач порушує умови використання системи.
+        </td>
     </tr>
      <tr>
         <td><b>Результат:</b></td>
-        <td></td>
+        <td>Заблокування користувача</td>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]: Better change to `Заблокований користувач`

File: docs/requirements/stakeholders-needs.md [hunk 4]
```diff
@@ -824,24 +872,37 @@
...
+         - BanUser_UserHasBeenBanned_EXC - користувача вже заблоковано</br>
+         - BanUser_CancelButton_EXC - користувач натиснув кнопку "Відміна"
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          1. Адміністратор фіксує підозрілу активність користувача<br/>
+          2. Адміністратор аналізує ситуацію<br/>
+          3. Адміністратор вирішує блокувати користувача, якщо справджується хоча б одна з передумов<br/>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]: Don't use if-cases in scenarios. Better to delete this step.

File: docs/requirements/stakeholders-needs.md [hunk 5]
```diff
@@ -857,24 +918,31 @@
...
     </tr>
      <tr>
         <td><b>Передумови:</b></td>
-        <td></td>
+        <td>Користувач заблокований</td>
     </tr>
      <tr>
         <td><b>Результат:</b></td>
-        <td></td>
+        <td>Розблокування користувача</td>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]: Change to "Розблокований користувач"

**Review by kol-oss — CHANGES_REQUESTED** [review]

File: docs/requirements/stakeholders-needs.md [hunk 6]
```diff
@@ -824,24 +872,37 @@
...
-        <td></td>
+        <td>Заблокування користувача</td>
     </tr>
      <tr>
         <td><b>Виключні ситуації:</b></td>
         <td>
+         - BanUser_NoMatchingUser_EXC - введені дані не відповідають жодному користувачеві</br>
+         - BanUser_UserHasBeenRemoved_EXC - користувача видалено з системи</br>
+         - BanUser_UserHasBeenBanned_EXC - користувача вже заблоковано</br>
+         - BanUser_CancelButton_EXC - користувач натиснув кнопку "Відміна"
```
Comment (mari4kaa) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
         - BanUser_CancelButton_EXC - адміністратор натиснув кнопку "Відміна"
```

File: docs/requirements/stakeholders-needs.md [hunk 7]
```diff
@@ -824,24 +872,37 @@
...
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          1. Адміністратор фіксує підозрілу активність користувача<br/>
+          2. Адміністратор аналізує ситуацію<br/>
+          3. Адміністратор вирішує блокувати користувача, якщо справджується хоча б одна з передумов<br/>
+          4. Адміністратор заповнює спеціальну форму для блокування, вказуючи причину та термін дії блокування<br/>
+          5. Система виконує блокування користувача і повідомляє його про це
```
Comment (mari4kaa) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
          4. Адміністратор заповнює спеціальну форму для блокування, вказуючи причину та термін дії блокування<br/>
          5. Адміністратор натискає кнопку "Підтвердити" (можливе BanUser_CancelButton_EXC)<br/>
          6. Система перевіряє валідність введених адміністратором даних (можливі BanUser_NoMatchingUser_EXC, BanUser_UserHasBeenRemoved_EXC, BanUser_UserHasBeenBanned_EXC)
          7. Система виконує блокування користувача і повідомляє його про це
```

File: docs/requirements/stakeholders-needs.md [hunk 8]
```diff
@@ -857,24 +918,31 @@
...
+         - UnbanUser_UserHasBeenUnbanned_EXC - користувача вже розблоковано</br>
+         - UnbanUser_CancelButton_EXC - користувач натиснув кнопку "Відміна"
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          1. Адміністратор фіксує потрібного користувача<br/>
+          2. Адміністратор натискає на "Розблокувати користувача" <br/>
+          3. Система виконує розблокування користувача і повідомляє його про це
```
Comment (mari4kaa) [inline docs/requirements/stakeholders-needs.md]: Add exceptions in brackets like i mentioned in previous comment

File: docs/requirements/stakeholders-needs.md [hunk 9]
```diff
@@ -319,24 +356,35 @@
...
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          <ol>
+            <li>Користувач обирає проєкт і натискає на кнопку "Видалити задачу"</li>
+            <li>Система відображає діалогове вікно для підтвердження операції видалення</li>
+            <li>Користувач натискає кнопку "Так"</li>
+            <li>Система видаляє завдання з проєкту</li>
```
Comment (mari4kaa) [inline docs/requirements/stakeholders-needs.md]: Add exceptions in brackets

**Review by mari4kaa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**